daftpunk/results.py

```diff
@@ -149,17 +151,12 @@ def get_page_url(self, offset):
         endpoint = "{}?{}".format(self.target, qstr)
         return endpoint
 
-    def get_me_stuff(self, url):
-
-        response = requests.get(url)
-        stuff    = None
+    def get_response(self, url):
+        return requests.get(url)
 
-        if self.mode == "soup":
-            stuff = bs4.BeautifulSoup(response.content, "html.parser")
-        elif self.mode == "response":
-            stuff = response
+    def get_soup(self, url):
+        return bs4.BeautifulSoup(requests.get(url), "html.parser")
```

Comment:
Makes sense, I forgot to access the content byte data here to

self.get_response(url).content